src/screens/Login/Login.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from "react";
+import { Text, View, Image } from "react-native";
+import AsyncStorageManager from "@storage/AsyncStorageManager";
+import { LoginBackground } from "@assets/images";
+import TextField from "@components/TextField";
+import GestureDetector from "@components/GestureDetector";
+import TextButton from "@components/TextButton";
+import textFieldStyles from "@components/TextField/styles";
+import styles from "./styles";
+
+const Login = ({ navigation: { replace } }: LoginScreenParamList) => {
```

Comment:
`LoginScreenParamList ` where is this defined?

Reply:
It is present in navigation declaration file
due to PR length i did not include that in this PR